generators/java/generator-utils/src/main/java/com/fern/java/utils/TypeReferenceUtils.java

```diff
@@ -183,7 +183,7 @@ public String visitLong() {
 
         @Override
         public String visitUint() {
-            return "Long";
+            return "Integer";
```

Comment:
I wonder if we should name this distinct even though it still maps to an `int`. That way if a user has an undiscriminated union with both there's no conflict. Otherwise the following type would have two things named with `Integer`:

```yaml
types:
  Number:
    discriminated: false
    union:
      - uint
      - integer
```